test/metabase/sample_data_test.clj

```diff
@@ -99,3 +101,42 @@
                  (hydrate :has_field_values)
                  (select-keys [:name :description :database_type :semantic_type :has_field_values :active :visibility_type
                                :preview_display :display_name :fingerprint :base_type])))))))
+
+(deftest write-rows-sample-database-test
+  (testing "should be able to execute INSERT, UPDATE, and DELETE statements on the Sample Database"
+    (with-temp-sample-database-db [db]
+      (mt/with-db db
+        (let [conn-spec (sql-jdbc.conn/db->pooled-connection-spec (mt/db))]
+          (testing "update row"
+            (let [quantity (fn []
+                             (->> (jdbc/query conn-spec "SELECT QUANTITY FROM ORDERS WHERE ID = 1;")
+                                  (map :quantity)))]
+              (testing "before"
+                (is (= [2]
+                       (quantity))))
+              (is (= [1]
+                     (jdbc/execute! conn-spec "UPDATE ORDERS SET QUANTITY = 1 WHERE ID = 1;")))
+              (testing "after"
+                (is (= [1]
+                       (quantity))))
+              ;; TODO: this shouldn't be necessary, since we're modifying a temp sample database.
+              (testing "restore"
+                (is (= [1]
+                       (jdbc/execute! conn-spec "UPDATE ORDERS SET QUANTITY = 2 WHERE ID = 1;"))))))
+          (let [rating (fn []
+                         (->> (jdbc/query conn-spec "SELECT RATING FROM PRODUCTS WHERE PRICE = 12.345;")
+                              (map :rating)))]
+            (testing "insert row"
```

Comment:
I think it's worth calling `(rating)` before the insert statement, just to highlight that the insert actually changed something.

Reply:
Done [Add test before inserting row](https://github.com/metabase/metabase/pull/28212/commits/c3b4925429a1a962f68c7de142ee8c17f2e7efd3)